modules/apps/captcha/captcha-rest-impl/rest-openapi.yaml

```diff
@@ -0,0 +1,119 @@
+components:
+    schemas:
+        Captcha:
+            properties:
+                captchaToken:
+                    type: string
+                image:
+                    type: string
+        FormCaptcha:
```

Comment:
I think CaptchaFormData would be better. What do you think?

Reply:
CaptchaForm is better